dataDispatcher.go

```diff
@@ -2,11 +2,11 @@ package indexer
 
 import (
 	"context"
-	"errors"
+	errorsGo "errors"
 	"sync"
 	"time"
 
-	"github.com/ElrondNetwork/elastic-indexer-go/client"
+	"github.com/ElrondNetwork/elastic-indexer-go/errors"
```

Comment:
please rename the package to `indexerError` or simply `error` and then `errorsGo` -> `errors`. Please apply this in the entire PR

Reply:
done. moved in root.